pkg/controllers/clusterexternalsecret/clusterexternalsecret_controller_test.go

```diff
@@ -361,31 +361,38 @@ var _ = Describe("ClusterExternalSecret controller", func() {
 				}
 			},
 		}),
-		Entry("Should crate an external secret and if one with the same name has been deleted", testCase{
+		Entry("Should crate an external secret if one with the same name has been deleted", testCase{
```

Comment:
Fix this flaky test, which seems to have the same problem with https://github.com/external-secrets/external-secrets/pull/2558 🙇 

https://github.com/external-secrets/external-secrets/actions/runs/5774855175/job/15652226003?pr=2582